openai-proxy/py_proxy/proxy.py

```diff
@@ -100,6 +100,15 @@ def proxy():
     url = request.json.get('url')
     if url.startswith('/'):
         source = url
+    elif url.startswith("s3://"):
+        trimmed_uri = url.replace('s3://', '')
+        # Split by '/' to separate bucket name and file key
+        split_parts = trimmed_uri.split('/')
+        bucket_name = split_parts[0]
+        file_key = '/'.join(split_parts[1:])
+        s3 = boto3.client('s3', 'us-east-1')
```

Comment:
Can we put `us-east-1` into a variable so we don't look like we're hard-coding it?  I can't remember if we have any sort of config available.

Reply:
Ack